contrib/generate_precompile.jl

```diff
@@ -0,0 +1,139 @@
+# Prevent this from being put into the Main namespace
+Base.reinit_stdio()
+let
+M = Module()
+@eval M begin
+julia_cmd() = (julia = joinpath(Sys.BINDIR, Base.julia_exename()); `$julia`)
+
+function generate_precompilable_package(path, pkgname)
+    mkpath(joinpath(path, pkgname, "src"))
+    write(joinpath(path, pkgname, "src", "$pkgname.jl"),
+          """
+          __precompile__()
+          module $pkgname
+          end
+          """)
+end
+
+function generate_precompile_statements()
+    t = time()
+    println("Generating precompile statements...")
+    println("──────────────────────────────────────")
+
+	# Reset code loading vars
+	push!(LOAD_PATH, "@stdlib")
```

Comment:
You've got a tab here and the line above it. It would be great if you could fix this the next time you push a commit.